src/App.js

```diff
@@ -28,10 +28,17 @@ const App = () => {
   // and trigger the ConfirmMonthModal if they are different
   const [newExpenseMonth, setNewExpenseMonth] = useState(selectedMonth);
 
+  // variables for toggling alert visibility and state
+  const [expenseWasAdded, setExpenseWasAdded] = useState(false);
+  const [expenseWasDeleted, setExpenseWasDeleted] = useState(false);
+  const [showExpenseAddedAlert, setShowExpenseAddedAlert] = useState(false);
+  const [showExpenseDeletedAlert, setShowExpenseDeletedAlert] = useState(false);
```

Comment:
these kind of state variables seem cumbersome, especially when we consider that there will be more alerts and states. can we think of something better here? (i have no ideas so far)

Reply:
Do you find it cumbersome that we use state variables at all or the fact that we need two per alert?

Reply:
two per alert. maybe a single variable which captures which alert(s) is/are open currently?
does it even make sense that multiple alerts/modals are open?

my guess is that if we simplify/unify modals & alerts, the code will become much simpler. (maybe only start simplifying alerts and go from there).

if you feel this makes sense, merge this MR first and then create a new one.